### PR TITLE
PLT-4609 - Fixing list modals on mobile

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
PLT-4609 - Fixing list modals on mobile

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-4609

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Added or updated unit tests (required for all new features)
- [ ] Added API documentation (required for all new APIs)
- [ ] All new/modified APIs include changes to the drivers
- [ ] Has enterprise changes (please link)
- [x] Has UI changes
- [ ] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/platform/blob/master/i18n/en.json) and [.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) updates
- [ ] Touches critical sections of the codebase (auth, upgrade, etc.)

### CODE REVIEW

File: webapp/components/searchable_user_list.jsx [hunk 1]
```diff
@@ -156,10 +156,7 @@ export default class SearchableUserList extends React.Component {
         }
 
         return (
-            <div
-                className='filtered-user-list'
-                style={this.props.style}
```
Comment (hmhealey) [inline webapp/components/searchable_user_list.jsx]: Why are you removing this prop? If some bad style is being passed in, we should remove it where it's being passed in, not by making the prop non-functional

Comment (asaadmahmood) [inline webapp/components/searchable_user_list.jsx]: Removed it from there as well @hmhealey

**Comment by jwilander** [comment]
ESLint errors
```
/var/lib/jenkins/jobs/mattermost-platform-pr/workspace/src/github.com/mattermost/platform/webapp/components/channel_invite_modal.jsx
  14:13  error  'Utils' is defined but never used  no-unused-vars

/var/lib/jenkins/jobs/mattermost-platform-pr/workspace/src/github.com/mattermost/platform/webapp/components/channel_members_modal.jsx
   17:13  error  'Utils' is defined but never used        no-unused-vars
  149:16  error  Block must not be padded by blank lines  padded-blocks

✖ 3 problems (3 errors, 0 warnings)
```

File: webapp/components/member_list_team.jsx [hunk 2]
```diff
@@ -122,7 +122,6 @@ export default class MemberListTeam extends React.Component {
 
         return (
             <SearchableUserList
-                style={this.props.style}
```
Comment (hmhealey) [inline webapp/components/member_list_team.jsx]: Thanks for changing those, but you missed one more. The style prop also needs to be removed from MemberListTeam's propTypes definition and the TeamMembersModal that uses this

**Pull request merged**